src/main/plugin/DtsPackPlugin.ts

```diff
@@ -216,7 +216,7 @@ export default class DtsPackPlugin {
 									size: () => src.length
 								};
 							})(r.files[file]);
-							compilation.assets[path.relative(opts.outDir, file)] = asset;
+							compilation.assets[file] = asset;

```

Comment:
The asset filename should be relative path (webpack seems to use `path.join` in Node.js), so we still need to use `path.relative`. (when `outDir` option is omitted, `file` would be an absolute path name.)

The base directory for relative path would be `webpackOutputPath || compilation.compiler.context`, so the code here might be:

```suggestion
							const assetBasePath = webpackOutputPath || compilation.compiler.context || '';
 							compilation.assets[path.relative(assetBasePath, file)] = asset;
```